tests/sources/test_cppcheck.py

```diff
@@ -165,7 +165,7 @@ def test_projects_with_cppcheck(self):
         skippedfiles = [
             'projects/robots/robotis/darwin-op/plugins/remote_controls/robotis-op2_tcpip/stb_image.h'
         ]
-        command = 'cppcheck --enable=warning,style,performance,portability --inconclusive --force -q '
+        command = 'cppcheck --enable=warning,style,performance,portability --inconclusive --force -q --library=qt '
         command += '--inline-suppr --suppress=invalidPointerCast --suppress=useStlAlgorithm -UKROS_COMPILATION '
         command += '--suppress=strdupCalled --suppress=ctuOneDefinitionRuleViolation --suppress=unknownMacro '
```

Comment:
This is not very consistent with the previous code where we put the space first and not last...

Reply:
I like putting the space first because it means this is a continuation of a previous string being build. Also putting the space first means we don't put it at the end which means we do not assume there will be some additional content.